### ISSUE TITLE
[Enhancement] Add option to only show failed rules with or without summary for those failed rules.

### ISSUE THREAD

**Issue by TheFlexican** [aws-cloudformation/cloudformation-guard#183]
**Is your feature request related to a problem? Please describe.**

Looking through the documentation, and help output from cfn-guard validate , I was unable to find my answer. We are using cfn-guard in the pipeline, and with version 1, it would only display rules that were non compliant. With version 2, I can only seem to suppress the summary of the checks?

When having multiple rules to check against a template, the output of such command can be overwhelming for our devops engineers to look trough to see where the failure is.

**Describe the solution you'd like**

Add option to the validation stage of cfn-guard to only display failed checks with or without their summary in the existing formats. If there are no failed rules exit 0

**Describe alternatives you've considered**

Using cfn-guard validate with json output and parse this output with jq.

**Comment by priyap286** [comment]
We recently made some improvements to the output of the `validate` command. These changes are available with the [most recent release](https://github.com/aws-cloudformation/cloudformation-guard/releases/tag/2.0.3). Please check the corresponding pull request (#154 ) where these changes were made to find more information on the improvements. I am closing this issue as we think the changes mentioned above solves your issue. Please reopen the issue if you still have unsolved issues with regard to this topic.

**Comment by TheFlexican** [comment]
Hi,

I'm using the latest version, but I'm not able to only show failed rules. 

What i would like to achieve, is run cfn-guard against a directory of rules, and the output should either show nothing when all rules have passed, or display the rules that have failed. If you think this is already achievable can u show me how to do it?

I can only come up with the below command, but this is not what I would like to see.

cfn-guard validate -d template.yaml -r ./rules/ --output-format single-line-summary --show-summary fail


Thank you :)

**Comment by TheFlexican** [comment]
What we did now, we created separate directories for critical and best practice rules. The output of both commands are in json and parsed by a self written Lambda. This lambda will a. Print out the error like version cfn-guard version 1 did b. based on the the type of rules it was scanning either exit with an error making codebuild stop, or exist nicely and let codebuild continue. For both actions the Lambda will use AWS SES with specific templates to inform the user who did the commit there were either errors or warnings with their template.
